app/src/main/java/org/gdg/frisbee/android/api/PlusImageUrlConverter.java

```diff
@@ -0,0 +1,41 @@
+package org.gdg.frisbee.android.api;
+
+import org.gdg.frisbee.android.api.model.plus.ImageInfo;
+import org.gdg.frisbee.android.app.App;
+
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class PlusImageUrlConverter implements Interceptor {
```

Comment:
http://gde-map.appspot.com/gde/list

now returns linkedin urls for social accounts instead of Google+. That's definitely a bug but we cannot trust it anymore. I suggest that we can disable GDE feature for this release. We can remove this ugly interceptor completely. 

And we can re-write GDE API later.